client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.html

```diff
@@ -734,6 +734,10 @@ <h4>{{ 'Amendments' | translate }}</h4>
 
         <!-- The HTML Editor for motions and traditional amendments -->
         <ng-container *ngIf="motion && editMotion && !motion.isParagraphBasedAmendment()">
+            <div *ngIf="getMotionEditConflictWarning()" class="alert-motion-edit-conflict">
+                {{ 'Warning: amendments exist for this motion. Editing this text will likely impact them negatively. Particularily, amendments might become unusable if the paragraph they affect is deleted.' | translate }}
+            </div>
+
```

Comment:
Instead, you might want to use the "raiseWarning" function from the base-view class.

https://github.com/OpenSlides/OpenSlides/blob/eadb0e2f0e28cd874b15c3cf91a299e5f86d5cb1/client/src/app/site/base/base-view.ts#L55-L57

Here is an example how to do so.
https://github.com/OpenSlides/OpenSlides/blob/eadb0e2f0e28cd874b15c3cf91a299e5f86d5cb1/client/src/app/site/motions/modules/motion-detail/components/motion-detail/motion-detail.component.ts#L1683-L1685

essentially, if there was a warning just fire the function. This will display a mat-snack bar containing the warning.
The only catch is that you have to do the translation in code (.ts)